lib/vtls/openssl.c

```diff
@@ -1936,7 +1936,15 @@ static void ssl_tls_trace(int direction, int ssl_ver, int content_type,
     }
     else
 #endif
-    {
+    if(content_type == SSL3_RT_CHANGE_CIPHER_SPEC) {
+      msg_type = *(char *)buf;
+      msg_name = "Change cipher spec";
+    }
+    else if(content_type == SSL3_RT_ALERT) {
+      msg_type = (((char *)buf)[0] << 8) + ((char*)buf)[1];
```

Comment:
remember to 'make checksrc':
~~~
./vtls/openssl.c:1944:50: warning: no space before asterisk (ASTERISKNOSPACE)
       msg_type = (((char *)buf)[0] << 8) + ((char*)buf)[1];
                                                  ^

Reply:
Ahh, I should have read the docs. Thanks for pointing that out! Fix pushed, and make checksrc now passes here.